backend/Repositories/SightingRepo.cs

```diff
@@ -51,5 +54,27 @@ public IEnumerable<Sighting> GetSightingsBySpeciesId(int speciesId)
                 .Where(s => s.ConfirmationStatus == ConfirmationStatus.Approved)
                 .OrderByDescending(s => s.SeenOn);
         }
+        public bool ConfirmRequest(int SightingId)
+        {
+            var sighting = _context.Sightings.Find(SightingId);
```

Comment:
I think I prefer the use of a `.Single()`:
```suggestion
            var sighting = _context.Sightings.Single(s => s.Id == SightingId);
```